README.md

```diff
@@ -102,6 +102,31 @@ Chose one of the available image types to build
 
 * `full`: include the full BISDN Linux system, including baseboxd and OF-DPA.
 
+> **_NOTE:_** The git CVE-2022-24765 directly affects all builds of BISDN Linux
+> prior to version 4.6.1, since those were release before the required patches
+> were released in yocto itself:
+>
+>   - https://git.yoctoproject.org/poky/commit/?h=dunfell&id=90cf135b046e4195512daeb8e8f516603d0f5d6c
+>   - https://git.yoctoproject.org/poky/commit/?h=dunfell&id=3a9cef8dbe3b5d81521536232c4ac67a9e81ac9a
+>   - https://git.yoctoproject.org/poky/commit/?h=dunfell&id=a75678145b35b3722f0cbba51aa699d3fd8f0bef
+>   - https://git.yoctoproject.org/poky/commit/?h=dunfell&id=a48231b5bfb5b1ca5de6dad6b3277a051eb6b55c
+>
+> Using a patched git version with an unpatched yocto one will cause builds to
+> fail with an error message like:
+>
+> ```
+> Summary: 1 task failed:
```

Comment:
Maybe add at least part of the actual error message here? So you can check that this is the right error you are looking at.

Reply:
I thought about it, but imho this part is the central message and contains most of the import key words.

Reply:
Yeah, but it doesn't say why it failed, or what part of the install failed. The install can fail for various reasons, so there really should be more context.

Reply:
done